src/main/java/frc/team2412/robot/subsystems/IntakeSubsystem.java

```diff
@@ -0,0 +1,125 @@
+package frc.team2412.robot.subsystems;
+
+import static frc.team2412.robot.Hardware.*;
+
+import com.revrobotics.CANSparkBase.IdleMode;
+import com.revrobotics.CANSparkFlex;
+import com.revrobotics.CANSparkLowLevel.MotorType;
+import edu.wpi.first.networktables.GenericEntry;
+import edu.wpi.first.wpilibj.shuffleboard.Shuffleboard;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import java.util.Map;
+
+public class IntakeSubsystem extends SubsystemBase {
+	// Constants
+	public static final double INTAKE_IN_SPEED = 0.3;
+	public static final double INTAKE_OUT_SPEED = -0.7;
+
+	public static final double INDEX_IN_SPEED = 0.3;
+	public static final double INDEX_OUT_SPEED = -0.3;
+
+	public static final double FEEDER_IN_SPEED = 0.3;
+	public static final double FEEDER_OUT_SPEED = -0.3;
+
+	public static final double SPEAKER_SHOOT_SPEED = 0.5;
+
+	// Motors
+	private final CANSparkFlex intakeMotorFront;
+	private final CANSparkFlex intakeMotorBack;
+	private final CANSparkFlex intakeMotorLeft;
+	private final CANSparkFlex intakeMotorRight;
```

Comment:
Intake motors are all Neo 550 motors, which use [CANSparkMax](https://codedocs.revrobotics.com/java/com/revrobotics/cansparkmax) instead of CANSparkFlex